site/content/device/ina219.md

```diff
@@ -0,0 +1,104 @@
++++
+title = "INA219"
+description = "High side current voltage and power sensor"
+picture = "/img/ina219.jpg"
++++
+
+# Overview
+
+[ina219](https://periph.io/x/periph/devices/ina219) provides support for a
+[popular power sensor from Texas Inbstruments](http://www.ti.com/product/INA219):
+
+# Features 
+* Senses Bus Voltages from 0 to 26 V
+* Reports Current, Voltage, and Power
+* 16 Programmable Addresses
+* High Accuracy: 0.5% (Maximum) 
+* Filtering Options
+* Calibration Registers
+
+![ina219](/img/ina219hookedup.jpg)
```

Comment:
You forgot to include the images in the commit?

Reply:
ahh.. just forgot to delete reference to my really bad photo. Adafruit has a good diagram.